### ISSUE TITLE
Merge my Japan NissanConnect implementation?

### ISSUE THREAD

**Issue by developerfromjokela** [dan-r/HomeAssistant-NissanConnect#32]
Hi.

I've forked your repo and implemented Japanese Domestic Market's NissanConnect app. Here's the repo: [https://github.com/developerfromjokela/HomeAssistant-NissanConnect-JP](https://github.com/developerfromjokela/HomeAssistant-NissanConnect-JP).

I think maybe if we somehow merge it into the main repo so we don't have two identical applications :D

- DFJ

**Comment by dan-r** [comment]
I agree it's a good idea to get it merged. Leave it with me and I'll merge your changes. 

Happy to make you a contributor for this repo too so you can maintain the Japan side

**Comment by dan-r** [comment]
@developerfromjokela I've made a start on the changes in the japan branch. Would you mind testing these?

https://github.com/dan-r/HomeAssistant-NissanConnect/tree/japan

**Comment by developerfromjokela** [comment]
> @developerfromjokela I've made a start on the changes in the japan branch. Would you mind testing these?
> 
> https://github.com/dan-r/HomeAssistant-NissanConnect/tree/japan

Hi. I haven't had a chance to test yet. I'll reply back when I can.

**Comment by dan-r** [comment]
Hey @developerfromjokela - appreciate its been a while but if this is still something you're using it'd be great to get it merged in!

**Comment by developerfromjokela** [comment]
> Hey @developerfromjokela - appreciate its been a while but if this is still something you're using it'd be great to get it merged in!

Hi. Honestly haven't been using anymore 😄 but should work I believe. If it's possible to sign in then should work. You can create an account at ev.nissanconnect.jp
